tests/fs/api.private.test.ts

```diff
@@ -0,0 +1,284 @@
+import * as fc from 'fast-check'
+
+import { IPFS } from 'ipfs-core'
+import { createInMemoryIPFS } from '../helpers/in-memory-ipfs'
+import FileSystem from '../../src/fs/filesystem'
+
+import * as ipfsConfig from '../../src/ipfs'
+import * as path from '../../src/path'
+import * as crypto from '../../src/crypto'
+
+
+let ipfs: IPFS | null = null
+let rootKey: string
```

Comment:
`rootKey` doesn't have to be global here, right? It's only used in `emptyFilesystem`, but never read.

Reply:
Yep, that's correct. Cleaning that up.